src/Home/Home.test.tsx

```diff
@@ -1,11 +1,14 @@
 import { fireEvent, render, screen } from "@testing-library/react";
 import Home from "./Home";
 import "@testing-library/jest-dom";
+import React, { useState } from "react";
 
-test("user's name is displayed on form submission", () => {
-    render(<Home />);
-
+test.skip("user's name is displayed on form submission", () => {
```

Comment:
you can actually fully delete this test as it's no longer the expected behaviour